explorer-backend/fly/guardiansets/gst_data.go

```diff
@@ -51,9 +45,21 @@ func NewGuardianSets(
 	}
 }
 
-func (gs *GuardianSets) GetGuardianSet(index int) (*common.GuardianSet, error) {
+func (gs *GuardianSets) GetGuardianSet(ctx context.Context, index int) (*common.GuardianSet, error) {
```

Comment:
It seems WH uses hardcoded guardian sets for mainnet. We may need to do the same later?

Reply:
Yes, we can also hardcode the guardian sets for mainnet.